src/snap/NotificationsEmitter.ts

```diff
@@ -0,0 +1,89 @@
+import { StrictEventEmitter } from "strict-event-emitter";
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import pEvent = require("p-event");
+import { DappeteerPage } from "../page";
+import * as dappeteer from "../../src";
+import { clickOnElement, openProfileDropdown } from "../helpers";
+import { NotificationItem, NotificationList } from "./types";
+
+interface EventsMap {
+  newNotification: (notification: NotificationItem) => void;
```

Comment:
```suggestion
  notification: (notification: NotificationItem) => void;
```
or `new` to make it simple.